File: LeetCode/412_FizzBuzz/fizz_buzz.py
def fizzBuzz(n: int) -> list[str]:

    solution_array = []

    for num in range(1, n+1):
        if num % 3 == 0 and num % 5 == 0:
            solution_array.append("FizzBuzz")
        elif num % 3 == 0:
            solution_array.append("Fizz")
        elif num % 5 == 0:
            solution_array.append("Buzz")
        else:
            solution_array.append(str(num))

    return solution_array

print(fizzBuzz(15))

File: LeetCode/412_FizzBuzz/fizz_buzz.js
export function fizzbuzz(number) {
  const result = [];

  for (let i = 1; i <= number; i++) {
    let output = "";

    if (i % 3 === 0) output += "Fizz";
    if (i % 5 === 0) output += "Buzz";

    const value = output || i;

    console.log(value);      // Ausgabe für dich
    result.push(value);      // Rückgabe für Tests
  }

  return result;
}